requirements.txt

```diff
@@ -259,3 +259,7 @@ flake8-tuple==0.2.12 \
 pycodestyle==2.0.0 \
     --hash=sha256:2ce83f2046f5ab85c652ceceddfbde7a64a909900989b4b43e92b10b743d0ce5 \
     --hash=sha256:37f0420b14630b0eaaf452978f3a6ea4816d787c3e6dcbba6fb255030adae2e7
+flask-debugtoolbar==0.10.0 \
```

Comment:
Should these be in a different requirements.txt (we don't we have a prod vs. pre-prod/dev list yet) lots of stuff going into a prod build that isn't needed potentially.